src/harness/sas_test_harness.py

```diff
@@ -452,6 +452,8 @@ def generatePpaRecords(ppa_records, cbsd_reference_ids):
     # Check cbsdReferenceID in ppaInfo and add cbsdReferenceID if not present.
     if not ppa_record['ppaInfo'].has_key('cbsdReferenceId'):
```

Comment:
It seems like there is no case in which we want to keep the cluster list (if present) which is already in the ppa_record, so perhaps these four lines should actually just be 

```ppa_record['ppaInfo'].update({'cbsdReferenceId': cbsd_reference_id})```

This would also be more consistent with the documentation for this function, and with how someone would naturally expect this function to work. What do you think?

Reply:
I agree with your explanation. 
Always updating the ppa cluster list in this function is more consistent with spec.
Thank you Kate for checking this!

Reply:
Thanks Kate, Dipti. I've added a commit to the PR reflecting the suggestion. Please review:
https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System/pull/859/commits/e4e157f3b8fc9f29cb6fbb283acc3a7cbb1212e1